hack/generate-swagger.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
Can we update this file name to `hack/update-swagger-docs.sh`? 
to stay in step with other scripts started with `hack/update-xxx.sh`.

Reply:
Ok.